packages/accordion/src/js/react.js

```diff
@@ -214,7 +215,7 @@ class AUaccordion extends React.PureComponent {
 					},
 					postfunction: function( target, state ) {
 						if( state === 'closed' ) {
-							target.style.display = '';
+							target.removeAttribute( 'style' );
```

Comment:
You don't want to do that cause there could be other styles applied by the user. That's why I removed only the explicit styles that were added

Reply:
Good point, i'll fix that up